sdk/scheduler/src/main/java/com/mesosphere/sdk/http/endpoints/DebugResource.java

```diff
@@ -24,11 +32,24 @@ public OfferOutcomeResource(OfferOutcomeTracker offerOutcomeTracker) {
      * @return HTML response of the table.
      */
     @GET
+    @Path("offers")
     public Response getOfferOutcomes(@QueryParam("json") boolean json) {
         if (json) {
             return ResponseUtils.jsonOkResponse(offerOutcomeTracker.toJson());
         } else {
             return ResponseUtils.htmlOkResponse(offerOutcomeTracker.toHtml());
         }
     }
+
+    /**
+     * Renders the current threads and thread state of the scheduler.
+     */
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("threads")
```

Comment:
Could add this to the API doc: https://github.com/mesosphere/dcos-commons/blob/master/docs/reference/swagger-api/swagger-spec.yaml#L644